### DISCUSSION TITLE
QST: visualize a region given a proj string

### DISCUSSION THREAD

**Discussion by marstonsward** [geopandas/geopandas#2734] [Q&A]
- [x] I have searched the [geopandas] tag on [StackOverflow](https://stackoverflow.com/questions/tagged/geopandas) and [GIS StackExchange](https://gis.stackexchange.com/questions/tagged/geopandas) for similar questions.

- [ ] I have asked my usage related question on [StackOverflow](https://stackoverflow.com) or [GIS StackExhange](https://gis.stackexchange.com).

```python
"test": {
      "projection": "+proj=laea +lat_0=0 +lon_0=-100 +units=m ",
      "xsize":750000.0,
      "ysize":750000.0,
      "pixelSize": 20000.0   
},
```
I would like to visualize this area using geopandas and create a geojson in both map coordinates and lat lon coordinates. 
Completely new to geopandas and can't really find a place to start. Can geopandas do this or do I need to look somewhere else?
Not sure if I'm asking the right question, but from what I see already in geopandas, I think this module can do this. Just need a place to start. Thanks.

**Comment by martinfleis** [comment]
I am not sure I follow your questions. Do you want to plot the bounds of the projection? Or transform your geometries to a projection and then plot it?

> Maybe someone can instead speak to how one can define a geographical region in geopandas from scratch?

What do you mean by a geographical region? A polygon with its boundaries? A projection?

**Comment by marstonsward** [comment]


**Reply by martinfleis** [reply]
Not entirely. 

> Define a geographical area using a projection such as Lambert Equal area

This is quite unclear. Lambert Equal Area is a way of transforming latitude and longitude coordinate into a plane, 2-dimensional space. There's no way to define an area using a projection. You can show the ideal bounds of the projection if it is aimed to be precise in one state or UTM zone, for example, but not of LEA because that is usually a global projection.

If you have a data in a projection A and want to turn them into a projection B, you can use `to_crs` in geopandas but I'm not sure that is what you're looking for.
